doc/SecurityProtocol.md

```diff
@@ -0,0 +1,153 @@
+# Tock Security Vulnerability Handling Process
+
+## Overview
+
+This document outlines internal procedures for handling security vulnerabilities in Tock.
+
+## Roles and Responsibilities
+
+### Triage Coordinator
+
+The Triage Coordinator (currently @charles37) responsibilities:
+
+- Initial review of security reports within 48 hours
+- Categorization and assignment of vulnerabilities
+- Coordination of response team
+- Communication with stakeholders
+- Tracking progress and ensuring resolution
+
+### Security Response Team
+
+Consists of subsystem maintainers called upon to address vulnerabilities.
+
+## Subsystem Contacts
+
+| Subsystem  | Primary Contact | Secondary Contact |
+| ---------- | --------------- | ----------------- |
+| Kernel     | TBD             | TBD               |
+| Drivers    | TBD             | TBD               |
```

Comment:
You can sign me up for Drivers if we're looking for volunteers. And/or libtock-c.